delivery_ups_oca/README.rst

```diff
@@ -38,16 +40,17 @@ Configuration
 To configure this module, you need to:
 
 #. Add options to your UPS carrier
+#. Go to *Inventory > Configuration* and check "Delivery Packages" option.
 #. Add a carrier account with delivery type ``ups`` and fill in your credentials
 
 Usage
 =====
 
-To use this module, you need to:
+You have to set Ups carrier in the stock picking you want to ship:
 
-#. Select the ups carrier on your sale orders or pickings
-#. Set exactly one delivery option
-#. Press the button to generate labels for a done picking
+ * In the stock picking form go to *Additional Info* tab and choose Ups as carrier and the service and product code. You only be able to choose this if the state of the picking is 'Ready to Transfer.
+
+ * When the picking is 'Transferred', it appears a *Create Shipping Label* button. Just push it, and if all went well the label will be 'attached'
```

Comment:
Put instructions for checking the delivery state (although I think there's a general cron for this, but it worths to mention all these aspects).

No automatic label generation? Isn't this an option on other carriers?

Mention also about cancelling.

Other thing: what about pricing?

Reply:
Some explanation added to clarify some of these things.

I don't understand about "_what about pricing?_". Can you get a example in some addon to understand it? (Actually, creation shipping process return price although we don't use in anywhere).

Reply:
On sales order, you are able to get the rate for the shipping. There's a method for that (not sure if `_rate_<carrier_code>` or similar)